docs_user/modules/proc_pulling-configuration-from-a-tripleo-deployment.adoc

```diff
@@ -82,8 +89,7 @@ services:
       ovn-ofctrl-wait-before-clear: edpm_ovn_ofctrl_wait_before_clear
 ----
 +
-This service is not an {rhos_prev_long} service executed in a container, so the description and the behavior is different. It is important to correctly configure an SSH config file or equivalent for non-standard services such as OVS. The `ovs_external_ids` does not run in a container, and the ovs data is stored
-on each host of our cloud: controller_1/controller_2/...
+You must correctly configure an SSH config file or equivalent for non-standard services, such as OVS. The `ovs_external_ids` service does not run in a container, and the OVS data is stored on each host of your cloud, for example, `controller_1/controller_2/` and so on.
```

Comment:
I have a question on all the text between step 3 and step 4: How much of this information is necessary for downstream customers to complete step 3?

It's too much text for a procedure, so we need to pare it down for the downstream docs.  We could choose the info that downstream customers need and conditionalize the rest for upstream docs.

Reply:
Well it describes a functionality which allow the user to pull configuration from other place than containers.
So maybe we can summerized a little or cut some of the explaination or as you suggest mention the upstream to get more information about it.

Reply:
@matbu I have an idea of how to revise this section, which I'll show in the next iteration of this PR.
In the meantime, can you clarify the following?

1. Can you clarify what you mean by "With the hosts key, os-diff loops on each host and runs the command in the service_command key"? Specifically, what is meant by "loops on each host"?
2. Can you clarify the sentence: "The service_command key provides the required information." Provides the required information to..? And what does "required information" refer to?